ViewPacker/tools/package.json

```diff
@@ -19,15 +19,15 @@
         "fs-extra": "^8.1.0",
         "glob": "^7.1.4",
         "mini-css-extract-plugin": "^0.8.2",
-        "node-sass": "^4.14.0",
-        "resolve-url-loader": "^3.1.0",
+        "node-sass": "^7.0.2",
```

Comment:
I believe we need to update some binaries but i don't know where, @alvesmiguel1 might know

Reply:
for the node-sass